sdk/common/build.gradle.kts

```diff
@@ -0,0 +1,82 @@
+plugins {
+    id("java-library")
+    id("maven-publish")
+
+    id("ru.vyarus.animalsniffer")
+}
+
+description = "OpenTelemetry SDK Common"
+extra["moduleName"] = "io.opentelemetry.sdk.common"
+
+val mrJarVersions = listOf(9)
+
+dependencies {
+    api(project(":api:all"))
+    api(project(":semconv"))
+
+    annotationProcessor("com.google.auto.value:auto-value")
+
+    testAnnotationProcessor("com.google.auto.value:auto-value")
+
+    testImplementation(project(":sdk:testing"))
+    testImplementation("com.google.guava:guava-testlib")
+}
+
+sourceSets {
+    main {
+        output.dir("build/generated/properties", "builtBy" to "generateVersionResource")
+    }
+}
+
+tasks {
+    register("generateVersionResource") {
+        val propertiesDir = file("build/generated/properties/io/opentelemetry/sdk/common")
+        outputs.dir(propertiesDir)
+
+        doLast {
+            File(propertiesDir, "version.properties").writeText("sdk.version=${project.version}")
+        }
+    }
+}
+
+for (version in mrJarVersions) {
```

Comment:
this also seems like a good candidate for extraction, especially if we're going to have more MR jars in the future.